tests/phpunit/tests/post/revisions.php

```diff
@@ -655,6 +655,51 @@ public function test_wp_save_post_revision_error() {
 		$this->assertWPError( $revision );
 	}
 
+	/**
+	 * Tests that wp_get_lastest_revision_id_and_total_count() returns last revision id and total count.
+	 *
+	 * @ticket 55857
+	 * @dataProvider data_wp_get_post_revisions_url
+	 */
+	public function test_wp_get_last_revision_id_and_total_count( $revisions ) {
+		$post_id = self::factory()->post->create();
+		wp_update_post(
+			array(
+				'ID'         => $post_id,
+				'post_title' => 'Some Post',
+			)
+		);
```

Comment:
```suggestion
		for ( $i = 0; $i < $revisions; ++$i ) {
			wp_update_post(
				array(
					'ID'         => $post_id,
					'post_title' => "Some Post $i",
				)
			);
		}
```

The update will still need to be looped to create the revisions.